boringtun/src/ffi/benchmark.rs

```diff
@@ -49,7 +49,7 @@ fn format_float(number: f64) -> String {
     formatted
 }
 
-fn run_bench(test_func: &mut FnMut() -> usize) -> f64 {
+fn run_bench(test_func: &mut dyn FnMut() -> usize) -> f64 {
```

Comment:
Why did you have to change this? We do not modify the boringtun directly, we fork it and do git subtree. Fork is here: https://github.com/kar/boringtun

Reply:
sorry about that. because the rustc warning I changed it to see if it'll work and it did. Noted